JiraPS/Public/New-JiraPermissionSchema.ps1

```diff
@@ -0,0 +1,65 @@
+function New-JiraPermissionSchema
+{
+    <#
+    .Synopsis
+
+    .DESCRIPTION
+       Create permission Schema
+    .EXAMPLE
+       New-JiraPermissionSchema
+    .EXAMPLE
+    .INPUTS
+    .OUTPUTS
+    #>
+    [CmdletBinding()]
+    param(
+        # Description of the permission Schema
+        [Parameter(Mandatory = $false)]
+        [String]
+        $Description,
+
+        # Name of the permission Schema
+        [Parameter(Mandatory = $true)]
+        [string]
+        $Name,
+
+        # JiraPS.PermissionSchema Object, Use Get-JiraPermissionSchema -ID $ID -Expand
+        [Parameter(Mandatory = $true)]
```

Comment:
Should support Pipeline.
Improve docs.
Maybe a more verbose name... `$InputSchema`?